libpod/container_internal_linux.go

```diff
@@ -574,6 +575,11 @@ func (c *Container) generateSpec(ctx context.Context) (*spec.Spec, error) {
 		g.AddProcessEnv("container", "libpod")
 	}
 
+	// Set notify
+	if c.config.SdNotifyMode == define.SdNotifyModeContainer {
+		g.AddProcessEnv("NOTIFY_SOCKET", "/run/notify/notify.sock")
```

Comment:
it is managed by the OCI runtime (both runc and crun do it).  `define.SdNotifyModeContainer` should simply propagate the `NOTIFY_SOCKET` env variable to the conmon process

Reply:
So, really this patch is not needed at all? Other then maybe adding /run/notify to the ignore list for diff?

Reply:
Again, incorrect, see below.

Container needs this env variable because OCI runtime will NOT have a NOTIFY_SOCKET env var.  We're circumventing OCI container notify socket support. (On purpose)

Reply:
The intent here is to work around the frankly broken behavior of OCI runtimes, which by standard do not exit until successfully notifying. This causes the OCI runtime to, during container start, hang around for potentially 10s of seconds - during which the container is locked by Podman to ensure that no other operations are triggered, and thus is unusable (will cause `podman ps` to hang, etc). We can't easily fix OCI runtimes, but we *can* work around them entirely.

Reply:
I see.  OK it makes sense.